prime-angular-frontend/src/app/modules/adjudication/pages/health-authorities/health-auth-care-types-page/health-auth-care-types-page.component.ts

```diff
@@ -43,7 +51,7 @@ export class HealthAuthCareTypesPageComponent implements OnInit {
       AdjudicationRoutes.HEALTH_AUTHORITIES,
       this.route.snapshot.params.haid
     ]);
-    this.filteredCareTypes = new BehaviorSubject<string[]>(this.configService.careTypes.map(ct => ct.name));
+    this.filteredCareTypes = new BehaviorSubject<HealthAuthorityCareTypeMap[]>(this.configService.careTypes.map(ct => { return { name: ct.name, code: ct.code } }));
```

Comment:
First off when wanting to return an object from an arrow function you can by just wrapping it in ( )
``` ct => ({ name: ct.name, code: ct.code }) ```

As well since CareTypeConfig is a subset of the fields in HealthAuthorityCareTypeMap
```this.configService.careTypes as HealthAuthorityCareTypeMap[]```

so we can just cast it, Also to make things more clear you could always define HealthAuthorityCareTypeMap like so
``` interface HealthAuthorityCareTypeMap extends CareTypeConfig ```

Reply:
Done, I created the CareTypeConfig in the config.model.ts file for now since we don't have that type config.